### ISSUE TITLE
Misleading error: Middleware stack must be array

### ISSUE THREAD

**Issue by colq2** [malijs/mali#94]
Hello,

I'm testing a bit around with Mali and I'm really like it so far.

There is one error which is maybe a bit misleading: `TypeError: Middleware stack must be an array!`

Finally I figured out, that it means, that there is a missing handler for a RPC call. But it took me a bit.

So for example we have (I skipped the messages): 

```
    syntax = "proto3";
    package users;
    
    service ManageUsers {
    	rpc Create (User) returns (Empty){}
    	rpc List (Empty) return (UserList){}
    }
```

And the simple implementation:

```
    const app = new Mali(PROTO_PATH, 'ManageUsers')
    
    function create(ctx){} // just mockup
    
    app.use('ManageUsers', 'create', create)
    
    app.start('0.0.0.0:50051')
    
    // TypeError: Middleware stack must be an array!
```

Maybe it could be better, when it would provide a warning like the grpc implementation:

```
    const grpc = require('grpc')
    let server = new grpc.Server()
    server.addService(services.ManageCoursesService, {
       create(){
         return {}
       }
    })
    
    // Method handler list for /users.ManageUser/List expected but not provided
```

I also tried to provide a default dummy function, but it don't work:

```
    app.use(() => {})
    
    // or
    
    app.use('ManageUsers', () => {})
    
    // Still TypeError: Middleware stack must be an array!
```

What do you think about the options to

1. Print warning instead of failing completely
2. Provide a dummy function for missing handlers (then it is possible to implement and test call after call)

**Comment by bojand** [comment]
Hello, this should be addressed in `0.15.0`. Everything is passed on to underlying grpc, so behaviour should match it now.

**Comment by colq2** [comment]
Works perfect now, great job!
